### PR TITLE
Remove msi builds on windows

### PR DESCRIPTION
**Pull request by Alphrag**
#### Pre-flight Checklist

1. Please remember that if you are logging a bug for some service that has _stopped working_ or is _working incorrectly_, please log the bug [here](https://github.com/ferdium/ferdium-recipes/issues)
2. If you are requesting support for a **new service** in Ferdium, please log it [here](https://github.com/ferdium/ferdium-recipes/pulls)
3. Please remember to read the [self-help documentation](https://github.com/ferdium/ferdium-app#troubleshooting-recipes-self-help) - in case it helps you unblock yourself for issues related to older versions of recipes that were installed on your machine. (These will get automatically upgraded when you upgrade to the newer versions of Ferdium, but to get new recipes between Ferdium releases, this documentation is quite useful.)
4. Please ensure you've completed all of the following.

- [x] I have read the [Contributing Guidelines](https://github.com/ferdium/ferdium-app/blob/develop/CONTRIBUTING.md) for this project.
- [x] I agree to follow the [Code of Conduct](https://github.com/ferdium/ferdium-app/blob/develop/CODE_OF_CONDUCT.md) that this project adheres to.
- [x] I have searched the [issue tracker](https://github.com/ferdium/ferdium-app/issues) for a feature request that matches the one I want to file, without success.

#### Description of Change

Delete the section that was building the `msi` installer on windows.

#### Motivation and Context

It has appeared recently that the `msi` builds have an inconsistent behaviour with the rest of the installers, meaning that the app doesn't appear with its correct version in the Windows Control Panel, and this behaviour can affect later updates that do not recognise the version is higher. Since they also do not have arm installer, it is preferable to use the full installer instead which contains all architectures.

#### Checklist

- [x] My pull request is properly named
- [x] The changes respect the code style of the project (`npm run prepare-code`)

#### Release Notes

Note: Remove the `msi` installers for Windows which caused problems

### CODE REVIEW

**Review by SpecialAro — COMMENTED** [review]
If we want to pursue this, maybe we should announce this change... This will break usability for users with the current msi version, am I correct? They won't be able to update without manually downloading and configure with the exe installer, right?

Should we try to make some counter measures for that before merging this PR?

(Nevertheless, my personal opinion is that this PR is really important, and we should merge it and get rid of the msi package entirely - at least until we find a way to fix inconsistencies).

**Comment by Alphrag** [comment]
tldr: This won't break anything. All updates will use the `exe` without the user noticing it (until they look at the control panel). So we can get rid of it without further thoughts.

Explanation: the way the auto-update works (for any OS) is that it fetches the file called `latest*.yml` on the release and use the url present in there. This means that for Windows, the auto-update feature *always* uses the `AutoSetup*.exe` file and never the `msi` one (on linux it is also the reason why only AppImages can auto-update). 
This is one of the other things showing that the way electron-builder works with the `msi` assets is problematic since it will be replaced by the operation of the `exe` installer at the first update. Currently, if you install through `msi` it creates a wrong entry in the Control Center, and then updating to the most up to date version creates another entry which now holds the correct informations. This should not happen! [I am planning to fill an issue on `electron-builder` directly as soon as I have time]

**Comment by Alphrag** [comment]
As we remove them, would it be worth exposing the `AutoSetup` installers for the different architecture separately, since the combined one is fairly heavy? The auto-updater will choose the one closer to the arch of the user, and thus save bandwidth. What do you think?

**Review by vraravam — APPROVED** [review]

**Comment by vraravam** [comment]
> As we remove them, would it be worth exposing the `AutoSetup` installers for the different architecture separately, since the combined one is fairly heavy? The auto-updater will choose the one closer to the arch of the user, and thus save bandwidth. What do you think?

If there's a simple way to do that, lets go for it. But, I would vote for whatever can unblock `6.0.0`

**Comment by vraravam** [comment]
@SpecialAro @Alphrag  - Should we merge this - and test out the effect using the next nightly?

**Comment by Alphrag** [comment]
> If there's a simple way to do that, lets go for it. But, I would vote for whatever can unblock `6.0.0`

Yep, the simple way is what I've just pushed: by forcing the artifact name to use the architecture, it separates all the installers.

**Comment by SpecialAro** [comment]
Thank you @Alphrag for your detailed explanation 😁

Then, we should go for it! Let's merge this PR

**Review by SpecialAro — APPROVED** [review]
